platform/accessibility/README.md

```diff
@@ -1,98 +1,63 @@
 
-**Teams need to fix all severity defect 0-4 accessibility issues**.
+All VA.gov products must legally comply with Section 508, and as such VFS teams need to fix all 508-defect issues.
 
 **Launch requirements:**<br/>
 VFS teams must fix[ 508-defect-0](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-0) and[ 508-defect-1](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1) issues before launching a Veteran-facing service. The DEPO Platform will enforce this minimum standard. <sup id="footnote1">[[1]](#f1)</sup>
 
 **Post-launch requirements:**<br/>
 VFS teams must iterate to resolve open[ 508-defect-2 through 508-defect-4 issues](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#post-launch-issues) to keep VA legally compliant. The DEPO Platform sends a public, weekly report to inform DEPO leadership of teams’ progress towards resolving issues.
 
-**So, what will VSP do to help you meet these requirements, and what are you expected to do?**<br/>
-Before requesting a staging review, VFS teams must do [foundational accessibility testing](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/staging-review-processes.md) and resolve or document any resulting issues. Be prepared to discuss your plan for when the unresolved issues might be addressed in the staging review meeting. 
-
-Teams with their own accessibility experts may be expected to do more robust testing beyond foundational accessibility tests. VSP will coordinate additional testing requirements on a case-by-case basis.
+**Read more about DEPO Platform accessibility standards**
+* [Defect Severity Rubric](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md)
 
-VSP will do a final check to review testing findings and comment on issues where additional guidance is helpful.
+## DEPO Platform and VFS accessibility roles and responsibilities
 
-## How to contact us
-* VSP Accessibility contact: Trevor Pierce, `@Trevor Pierce` in Slack, `1Copenut` in Github
-* Slack channels: [#vetsgov-accessibility](https://dsva.slack.com/channels/vetsgov-accessibility)
-* Github labels: `508/Accessibility`
+Veteran-facing services (VFS) teams are responsible for the accessibility of their products. The Digital Experience and Product Office (DEPO) Platform supports their work at scale by creating and maintaining VA.gov accessibility standards required for production and by providing guidance as VFS teams work to meet those standards. Code quality -- including meeting overall accessibility standards (including both Section 508 and selected WCAG recommendations) -- is the responsibility of each individual VFS team.
```

Comment:
What do you think about splitting the parenthetical statement in the last sentence out to a complete sentence of its own? I want to be very clear with teams what the standard for compliance is. Maybe something like:

> Code quality -- including meeting overall accessibility standards  -- is the responsibility of each individual VFS team. Code must pass the [WCAG 2.0](https://www.w3.org/TR/WCAG20/) Level A and AA success criteria to comply with Section 508.

Reply:
I updated to match the language on the roles and responsibilities document: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/roles-and-responsibilities/accessibility.md